build/release/root/etc/notices/COPYRIGHT

```diff
@@ -1,5 +1,6 @@
 Open Maryland, not Indiana Operating System (OmniOS)
 Copyright 2016 OmniTI Computer Consulting, Inc. All rights reserved.
+Copyright 2017 OmniOS Community Edition Association (OmniOSce)
```

Comment:
That wouldn't fit on a single line. I could put the `All rights reserved.` on the following line though.